src/gpl/src/mbff.h

```diff
@@ -0,0 +1,77 @@
+#include <vector>
```

Comment:
Try running clang format on these files. The indenting is using tabs, and not spaces.